brewtils/rest/client.py

```diff
@@ -142,6 +142,10 @@ def __init__(
         if client_cert:
             self.session.cert = client_cert
 
+        namespace = kwargs.get("namespace", None)
+        if namespace:
+            self.session.headers.update({"bg-namespace": namespace})
```

Comment:
I guess this makes sense for querying on particular namespaces.

For doing `POST`, or `PUT` operations, are we expecting the namespace to be in the body?

Reply:
Right now, no. I have branches (brewtils and bg-utils) where I've added a namespace field to the models, but that hasn't been merged. If / when that happens then yeah, this could be tricky: what happens if the header and the body namespace conflict. :man_shrugging: